docs/Spark/gems/transform/fuzzy-match.md

```diff
@@ -0,0 +1,47 @@
+---
+title: FuzzyMatch
+id: fuzzy-match
+description: Identify non-identical duplicates in your data
+tags:
+  - gems
+  - fuzzy match
+---
+
```

Comment:
which dependencies are needed? (selfish question, I want to create an "example" for this gem)

Reply:
bloop - I am looking for the SQL dependency. So this question is not relevant for this PR.